mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB = "$(mktemp --suffix -snapshot.db)"
 
 ## Do the slooow thing of snapshotting the database
 ./manage.py snapshot_public_data | gzip > "$TEMPFILE"
 
+## Create a new settings to specify the name of the temporary snapshot db
+touch mysite/snapshot_settings.py
+echo "from mysite.settings import *; DATABASES = {
+    'default': {
+        'NAME': os.path.join(MEDIA_ROOT_BEFORE_STATIC, "$TEMPDB"),
+        'ENGINE': 'django.db.backends.sqlite3',
+        'CHARSET': 'utf8',
+    },
+}" > mysite/snapshot_settings.py
+
+# Load in the snapshot data, which automagically generates the temporary db in root directory
+echo "TEMPFILE VARIABLE: " $TEMPFILE
+./manage.py syncdb --noinput --migrate --settings=mysite.snapshot_settings
+./manage.py migrate --settings=mysite.snapshot_settings
+./manage.py loaddata $TEMPFILE --settings=mysite.snapshot_settings
+
 # Set the permissions so that, after the file gets pushed to the web,
 # the web server permits people to download these snapshots.
 chmod 644 "$TEMPFILE"
+chmod 644 "$TEMPDIR"
 
-## push this somewhere so that we save it
+## push this somewhere so that we save it. Do the same thing for the db file.
 rsync -q "$TEMPFILE" inside@"$HOST":/var/web/inside.openhatch.org/snapshots/$(date -I).json.gz
+rsync -q "$TEMPDB" inside@"$HOST":/var/web/inside.openhatch.org/snapshots/$(date -I).db
 
-## finally, delete the local temporary file.
+## finally, delete the local temporary file, the local temporary db, and the local snapshot settings
 rm "$TEMPFILE"
+rm "$TEMPDB"
+rm snapshot_settings.py
```

Comment:
since you're creating `$SNAPSHOT_SETTINGS_PATH` and it's just a string, you can put whatever you want in it. If it's a relative path, rather than an absolute one, that's fine as long as you remember and are consistent.